app_server/app/models/Storage.js

```diff
@@ -77,37 +79,48 @@ Class.createUser = function(particulars) {
 
 /**
  * @param  {string} email - the user's email
- * @return {Promise<Sequelize.object>}
+ * @return {Promise<Sequelize.object> | False}
  */
 Class.getUserByEmail = function(email) {
   return this.models.User.findOne({
     where: {
       email: email
     }
-  })
-    .catch(function(err) {
-      logger.error('Unable to retrieve user');
+  }).then(function(res) {
+    if (res === null) {
+      logger.info('No such user');
       return false;
-    });
+    } else {
+      return res;
+    }
+  }).catch(function(err) {
+    logger.error('Error in retrieving user');
+    return false;
+  });
```

Comment:
If you are logging errors with an error object provided, it's better to log the error object itself or the error message from the error object too e.g. `logger.error('Error in retrieving user: %j', err)` or `logger.error('Error in retrieving user: %s', err.message)` etc.